deps/rabbitmq_prometheus/Makefile

```diff
@@ -9,7 +9,7 @@ endef
 PROJECT := rabbitmq_prometheus
 PROJECT_DESCRIPTION = Prometheus metrics for RabbitMQ
 PROJECT_MOD := rabbit_prometheus_app
-DEPS = accept cowboy rabbit rabbitmq_management_agent prometheus rabbitmq_web_dispatch
+DEPS = accept cowboy rabbit rabbitmq_management rabbitmq_management_agent prometheus rabbitmq_web_dispatch
```

Comment:
This means that if you enable the Prometheus plugin, the management plugin must be enabled (so that its modules are on the code path).

A lot of people use Prometheus and specifically do not want management to be enabled.

Reply:
Ah, good point! But still OK to use it as a 'build dependency' right, as it will use mgmt auth functions (which should still be in the release right, even though the plugin is disabled?)